### ISSUE TITLE
Detach, Folders, and Outfits, Oh My !!

### ISSUE THREAD

**Issue by SilkieSabra** [OpenCollarTeam/OpenCollar#155]
Here we have a function train wreck.  I am learning all the time and didn't see this coming when nirea made that wonderful Detach feature.

There are three ways that users can protect worn items from being lost during RLV dress and undress.  1) Using a .hidden folder; 2)  applying a (nostrip) tag to the end of the item name or its folder; and 3) moving the item into a folder in Outfits named .core.  

If you use hidden folders by putting a dot in front of the name that protects the items from being seen in the Folders menu.  But they can still be seen, and detached, from the Detach list.

If you add the (nostrip) tag to the end of the item name or folder it's in directly, it will be seen in Detach but the Detach command will not execute.

.core is nearly useless, unless there is some item you ALWAYS wear in every outfit and NEVER include it in any of your Outfits in .outfits';  because Outfits uses Replace, and .core force wears everything in the folder every time you use the Outfits function.  My collar is the only thing i have like that, other than required system layers which can't be doubled.  And my collar is already locked anyway.  But if my collar, or say my mesh body, is in the Outfit, and also in .core, I will end up wearing two copies.

(nostrip) conflicts with Outfits in a similar way to .core.  If you have an item set up with a (nostrip) tag it will not be detached during the Replace.  Suppose you don't want your favorite hair stripped.  You put it  in any hidden folder.  That hides it from Folders, but it can still be  Detached.  So you add a (nostrip) tag.  That prevents it from being detached.  It *also* prevents it from being detached when an Outfit is Replaced.  Again you end up wearing two hairs; or worse, two copies of the same mesh body; the variations are endless.

Bottom line is you can optimize your folder tree for Outfits or for Detach, but not both.

So for the next release can we please do a massive overhaul and rethink how we handle Outfits, and Detach?  there must be some way to get it so it all works together. 

Thanks to Ambyr Moonwall and Loki Bluebird for their help, Ambyr called my attention to the conflicts and both Ambyr and Loki helped test.

**Comment by Medea-Destiny** [comment]
The .core functionality would be a lot more useful if the folder was simply locked before anything else, so that nothing that's inside it can be removed. That would allow for items to be put in .core that are not always worn, but shouldn't be detached if they are worn.

What I'm not yet sure (need to do some experiments) is what happens if you attempt to detach a folder that contains an inventory link to an item in a locked folder, which would have an impact on how we can use the .core for an extended nostrip functionality.

I think we also need to have attachment point filtering for the detach function. For example, you could set up the detach function to not list anything attached to the skull attachment point, to avoid it being used to remove a hairstyle.
